_cities/leipzig.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Leipzig
+location: Leipzig, Germany
+tagline: Coding without the hype
+twitter: ots_le
+members:
+  - norma: Norma
+  - christina: Christina
+  - krissi: Krissi
+---
+
+## About
+
+Norma and Christina started "LearnDjango SelbsthilfeTreff" beginning of 2018 and are now opening up this meetup to all learners and all languages. Together with Krissi we offer a biweekly meetup to learn, collaborate and meet other learners at Basislager in Leipzig. 
+
+We encourage women of all ages to join us.
```

Comment:
Can you formulate this sentence slightly more open to people of other genders? Something like
"We encourage learners of all ages to join us, and welcome women and non-binary people in particular."?